tests/network-tests/src/flows/storage/initDistribution.ts

```diff
@@ -1,37 +1,40 @@
-import { FlowProps } from '../../Flow'
-import { extendDebug } from '../../Debugger'
 import {
   DistributionBucketFamilyMetadata,
   DistributionBucketOperatorMetadata,
   IDistributionBucketFamilyMetadata,
   IDistributionBucketOperatorMetadata,
 } from '@joystream/metadata-protobuf'
-import { CreateInterface, createType } from '@joystream/types'
-import { BagId, DistributionBucketFamilyId, DynamicBagId, StaticBagId } from '@joystream/types/storage'
+import { createType } from '@joystream/types'
+import { DistributionBucketFamilyId, WorkerId } from '@joystream/types/primitives'
 import { SubmittableExtrinsic } from '@polkadot/api/types'
+import {
+  PalletStorageDynamicBagIdType as DynamicBagId,
+  PalletStorageStaticBagId as StaticBagId,
+} from '@polkadot/types/lookup'
 import _ from 'lodash'
+import { extendDebug } from '../../Debugger'
+import { FlowProps } from '../../Flow'
 import { Utils } from '../../utils'
-import { WorkerId } from '@joystream/types/working-group'
 
 type DistributionBucketConfig = {
   metadata: IDistributionBucketOperatorMetadata
-  staticBags?: CreateInterface<StaticBagId>[]
+  staticBags?: StaticBagId[]
   operatorId: number
 }
 
 type DistributionFamilyConfig = {
   metadata?: IDistributionBucketFamilyMetadata
   buckets: DistributionBucketConfig[]
   dynamicBagPolicy: {
-    [K in keyof typeof DynamicBagId.typeDefinitions]?: number
+    [K in DynamicBagId['type']]?: number
   }
 }
 
 type InitDistributionConfig = {
   families: DistributionFamilyConfig[]
 }
 
-export const allStaticBags: CreateInterface<StaticBagId>[] = [
+export const allStaticBags = [
```

Comment:
You can use `CreateInterface<StaticBagId>[]` (again) to avoid the type assertions below (`(allStaticBags as unknown) as StaticBagId[]`) and use `allStaticBags.map((sb) => createType('PalletStorageStaticBagId', sb))` instead

Reply:
Addressed in [b552fba](https://github.com/Joystream/joystream/pull/3997/commits/b552fba6a40a75129fb358c4b771150313077472)